kubernetes/helm/prometheus-operator/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+appVersion: "1.0"
```

Comment:
This should be what the `prometheus-operator` version is, IMO.

Reply:
Matches stable/prometheus-operator now
`$ helm inspect stable/prometheus-operator --version 8.3.3 | grep appVersion
appVersion: 0.34.0`